pkgs/build-support/build-bazel-package/default.nix

```diff
@@ -4,10 +4,20 @@
 , lib
 }:
 
-args@{ name, bazelFlags ? [], bazelBuildFlags ? [], bazelFetchFlags ? [], bazelTarget, buildAttrs, fetchAttrs, ... }:
+args@{
+  name
+, bazelFlags ? []
+, bazelBuildFlags ? []
+, bazelFetchFlags ? []
+, bazelTarget
+, buildAttrs
+, fetchAttrs
+, removeRulesCC ? true
```

Comment:
Can you document this argument, please? it seems to be a bit specific to the gvisor use-case?

Reply:
Done in #74013